package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "moments": "0.0.2",
```

Comment:
this may be a typo :) I think you mean moment.

This is a real npm module! And it has `moment` as a dependency - this is why your code is still working correctly but in production it is important to make sure the correct modules are loaded.